### ISSUE TITLE
Where we call 'loss.backward()'

### ISSUE THREAD

**Issue by 9p15p** [open-mmlab/mmsegmentation#221]
In my own code's train function,
I will call `loss.backward()` for backpropagation.

But I can't find any place in MMsegmentation, we call this function,
could you please explain it ?

Thank you.

The 'loss.backward()' isn't be used.
![image](https://user-images.githubusercontent.com/40333030/97782209-3bc4db00-1bcb-11eb-8154-b4174117194f.png)

**Comment by 9p15p** [comment]
I check MMCV's docs，there's a hook called 'OptimizerHook', but I don't find it in MMsegmentation, too.

So, where do we  do backpropagation for loss?
In usual train.py, we always use the following code to update parameters.

```
        optimizer.zero_grad()
        # backward pass
        loss.backward()
        # update parameters
        optimizer.step()
```

**Comment by YLyeliang** [comment]
Maybe you should check mmcv/runner/hooks/Optimizer.py. Times have changed, my lord.
